src/ResourceManager/Insights/Commands.Insights/ChangeLog.md

```diff
@@ -22,6 +22,39 @@
 ## Version 5.0.1
 * Fixed formatting of OutputType in help files
 
+* Using Microsoft.Azure.Management.Monitor SDK 0.19.1-preview
```

Comment:
@pkamath13 @gucalder you will need to move this content to be under the `## Current Release` header